quadproduction/OpenPype
# Bugfix: hide deleted projects from ftrack on OP

## Changelog Description
Create a custom action to hide projects on OP that are deleted on ftrack or that doesn't have an ftrackId in mongo.

## Testing notes:
1. open ftrack in webbrowser
2. go in Overview -> Projects
3. go to Actions -> OpenPype Admin and choose "OpenPype Admin - Hide deleted projects"
4. go to OP project settings and see the hidden projects
